pegasus/test/fixtures/fake_dashboard.rb

```diff
@@ -73,6 +73,29 @@ module FakeDashboard
   ]
 
   #
+  # Fake Data: Courses
+  #
+  COURSES = [
+    {
+      id: 15,
+      name: 'csp',
+      created_at: '2016-01-01 00:01:02',
+      updated_at: '2016-01-01 00:01:02'
+    }
+  ]
+
+  #
+  # Fake Data: Scripts
+  #
+  SCRIPTS = [
+    {
+      id: 10,
+      name: 'flappy',
+      hidden: 0
+    }
+  ]
```

Comment:
When assigning a script to a section, we check to see that it's a valid one. If the script does not exist in our db, it's considered invalid (thus tests will fail without this script in our fake db).